### PR TITLE
Color filter metadata

### PR DESCRIPTION
**Pull request by nwtn**
This PR adds CSS `filter` information to the token metadata. This data can be used to set colors of, e.g., SVGs rendered in `` elements.

I added the filter information to the Invision DSM in the description field, as YAML Front Matter. Currently the DSM API is not returning this field, so the metadata is also stored locally in a new `data/` dir.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -93,6 +93,7 @@
     "theo": "8.0.0-beta.2",
     "tinycolor2": "^1.4.1",
     "xml": "^1.0.1",
+    "yaml-front-matter": "^4.0.0",
```
Comment (kaelig) [inline package.json]: Can we use https://www.npmjs.com/package/gray-matter instead? That's what we've been using on other `polaris-*` repos, might as well reuse the same thing. (also: it seems more popular)

**Comment by kaelig** [comment]
Code looks good!

Can you add some documentation on how you'd leverage those values (one or two use cases would be great)?

**Comment by nwtn** [comment]
> Can you add some documentation on how you'd leverage those values (one or two use cases would be great)?

Great idea. Do you have a strong opinion about the right place to put those? Most obvious place to me seems like the `Usage` section of the README, but I’ll defer to you @kaelig.

**Comment by kaelig** [comment]
> Do you have a strong opinion about the right place to put those? 

No strong opinions, https://github.com/Shopify/polaris-tokens#usage seems like a good start.

---

Can you also document how those colors were generated, in case anyone wants to add additional ones?

File: scripts/get-tokens-from-invision-dsm.js [hunk 2]
```diff
@@ -20,6 +21,11 @@ const fetchTokens = async () => {
 // Invision’s API returns colors under this path
 const getColorTokens = (object) => object.list.colors[0].colors;
 
+// local metadata
```
Comment (kaelig) [inline scripts/get-tokens-from-invision-dsm.js]: Can you mention that this is a temporary workaround until DSM is resurfacing the `description` in its API?

File: README.md [hunk 3]
```diff
@@ -115,6 +115,73 @@ polaris_colors = JSON.parse(polaris_token_file)
...
+  No background, red filter
+</div>
+
+<div style="background-color: #fff; filter: brightness(0) saturate(100%) invert(28%) sepia(67%) saturate(3622%) hue-rotate(353deg) brightness(89%) contrast(95%)">
+  White background, red filter
+</div>
+```
+
+<div>No background, no filter</div>
+<div style="background-color: #fff">No background, no filter</div>
```
Comment (kaelig) [inline README.md]: These inline styles don't seem to render in GitHub 😞

**Comment by nwtn** [comment]
i think this is good to go, other than some squashing/fixup-ing (which i’ll do now)

**Pull request merged**